test/features/binary_build.feature

```diff
@@ -17,3 +17,17 @@ Feature: Use binary build to deploy the service
 
     Then Kogito application "process-springboot-example" has 1 pods running within 5 minutes
     And HTTP GET request on service "process-springboot-example" with path "orders" is successful within 2 minutes
+
+#####
+
+  @quarkus
+  Scenario: Deploy process-quarkus-example using binary build
```

Comment:
What about native mode ? :)

Reply:
Ops, sorry. Moving back the ticket to In Progress.

Reply:
Updated with native support